crates/config/src/zksync.rs

```diff
@@ -103,8 +103,10 @@ impl ZkSyncConfig {
             detect_missing_libraries: self.detect_missing_libraries,
             system_mode: self.eravm_extensions,
             force_evmla: self.force_evmla,
-            // TODO: See if we need to set this from here
-            output_selection: Default::default(),
+            output_selection: serde_json::from_str(
+                "{\"*\":{\"*\":[\"abi\",\"evm.methodIdentifiers\",\"metadata\", \"irOptimized\"]}}",
```

Comment:
Add a comment specifying that we specify it here so we can retrieve the ABI also. We should verify if we need the metadata or what